main.go

```diff
@@ -809,29 +805,99 @@ func NoteActionShow(noteID string) {
 	if _, err := tuneApp.GetNoteByID(noteID); err != nil {
 		errorExit("%v", err)
 	}
-	fileName := fmt.Sprintf("%s%s", NoteTuningSheets, noteID)
-	if _, err := os.Stat(fileName); os.IsNotExist(err) {
-		_, files := system.ListDir(ExtraTuningSheets, "")
-		for _, f := range files {
-			if strings.HasPrefix(f, noteID) {
-				fileName = fmt.Sprintf("%s%s", ExtraTuningSheets, f)
-			}
-		}
-		if _, err := os.Stat(fileName); os.IsNotExist(err) {
-			errorExit("Note %s not found in %s or %s.", noteID, NoteTuningSheets, ExtraTuningSheets)
-		} else if err != nil {
-			errorExit("Failed to read file '%s' - %v", fileName, err)
-		}
-	} else if err != nil {
-		errorExit("Failed to read file '%s' - %v", fileName, err)
-	}
+	fileName, _ := getFileName(noteID)
 	cont, err := ioutil.ReadFile(fileName)
 	if err != nil {
 		errorExit("Failed to read file '%s' - %v", fileName, err)
 	}
 	fmt.Printf("\nContent of Note %s:\n%s\n", noteID, string(cont))
 }
 
+// NoteActionDelete deletes a custom Note definition file and
+// the corresponding override file
+func NoteActionDelete(noteID string) {
+	if noteID == "" {
+		PrintHelpAndExit(1)
+	}
+	if _, err := tuneApp.GetNoteByID(noteID); err != nil {
+		errorExit("%v", err)
+	}
+
+	txtConfirm := fmt.Sprintf("Do you really want to delete Note (%s)?", noteID)
```

Comment:
@angelabriel I think it would make sense to avoid the doubled confirmation here. As you will ask a second more specific confirmation anyway below before deleting, I would do the checks, and ask only the more specific confirmation to the user.

Reply:
I only ask one time for a confirmation (with readYesNo). 'txtConfirm' is the variable, which contains the text for the confirmation question.  So 'txtConfirm :='  is only the variable declaration, so that I can simply change the text in the following 'if then else' part to the more specific confirmation text.
But if this is to confusing, I can simply declare it as an empty variable like 'txtConfirm := ""' or  as 'var txtConfirm string'

Reply:
Oh, my bad. That is fine.. it was a fast code reading... sorry...